### DISCUSSION TITLE
How to prevent Google captch within using Hiddify? چگونه با استفاده از هیدیفای از کپچای گوگل جلوگیری کنیم؟

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#486] [Q&A سوالات رایج]
احتمالا هنگام استفاده از VPN به این مشکل برخوردید که الگوریتم تشخیص ربات گوگل فعال می شود و هنگام جستجو به شما کپچایی نمایش می دهد که باید آن را پاس کنید.

![photo_2023-03-19_11-54-07](https://user-images.githubusercontent.com/125398461/226162984-e5ccfde6-4fa0-4c27-a4ee-2ec542b14172.jpg)

پس از پاس کردن تیک سبز رنگ زده می شود و اجازه جستجو داده شود. 

![Screenshot_20230319_114813](https://user-images.githubusercontent.com/125398461/226163009-9a181a34-7c78-4dc5-8437-4032574b62ef.png)

این کار بعضی وقت ها آزاردهند می شود. برای رفع این مشکل چه راه کاری وجود دارد؟

**Comment by lymanjre** [comment]
اصولا این مشکل به دلیل حساسیت گوگل به استفاده ربات ها از این موتور جستجو اتفاق می افتاد. همچنین اگر گوگل رفتا مشکوکی مشاهده کند که مشابهتی با رفتار یک ربات داشته باشد؛ کپچا را فعال می کند.

وقتی از یک سرور VPN استفاده می شود؛ کاربران به صورت همزمان از یک IP ثابت چندین درخواست جستجو به گوگل ارسال می کنند و این رفتار شبیه به رفتار یک ربات است. به همین دلیل کپچا فعال می شود.

راه کار: 
اگر به پایین پیغام دقت کتید؛ می بینید که آیپی ای که شما با آن به سمت سرورهای گوگل آمده اید نمایش داده می شود. در اغب مواقع روی IP ورژن ۶ کپچا ظاهر می شود.

شما می توانید با غیرفعال کردن IPv6 روی سرور خود این مشکل را حل نمایید.

## غیرفعال کردن IP ورژن ۶ روی هیدیفای:
برای این کار به بخش تنظیمات بروید؛ در قسمت تنظیمات عمومی گزینه ` فقط آیپی ورژن ۴ ` را فعال کنید.


![Screenshot_20230319_230930](https://user-images.githubusercontent.com/125398461/226204875-2f8bfee4-90df-4c2f-87b0-b37143b58619.png)

سپس ثبت و اعملا تغییرات را بزنید.

## غیر فعال کردن IP ورژن ۶ روی سرور:


### غیرفعال کردن IP ورژن ۶ روی CDN:
اگر از CDN استفاده می کنید نیاز دارید که رکوردهای DNS ثبت شده خود را حذف نمایید. برای این کار وارد کلادفلر شوید و در قسمت DNS مربوط به دامنه خود رکوردهای AAAA را حذف نمایید.
![222444012-2fa4a2c2-ff89-493e-b92c-01a26d7788b7](https://user-images.githubusercontent.com/125398461/226206615-b3fff14f-7ea2-4169-ac5c-02b36881590d.png)

**Comment by mehrdadtopme** [comment]
سلام من فقط از ای پی ۶ استفاده میکنم اما هنوز هم برای من این پیغام میاد
راهی هست که این پیغام دیگه نیاد؟
